frontend/src/utils/keycloakHook.js

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useRef } from 'react';
+import { useKeycloak } from '@react-keycloak/web';
+
+const useCustomKeycloak = ({ customNonce } = {}) => {
```

Comment:
I think you could set the default up here while destructuring: 

`{customNonce = createUUID()}`

Reply:
will do!